egs/avspeech/local/loader/download.py

```diff
@@ -0,0 +1,76 @@
+import os
+import time
+import tqdm
+import argparse
+import subprocess
+import pandas as pd
+from pathlib import Path
+import concurrent.futures
+
+from constants import VIDEO_DIR
+
+def download(link, path, final_name=None):
+    command = "youtube-dl {} --output {}.mp4 -f 'bestvideo[ext=mp4]+bestaudio[ext=m4a]/mp4'"
```

Comment:
```suggestion
command = "youtube-dl {} --no-check-certificate --output {}.mp4 -f 'bestvideo[ext=mp4]+bestaudio[ext=m4a]/mp4'"
This would fix the download error I got at first
```

Reply:
Can you comment the last line so I can commit from here?